README.md

```diff
@@ -2,19 +2,49 @@
 This is a machine code simulator coded in Python 3 based upon the 
 [Brookshear Machine](https://w3.cs.jmu.edu/cs101/unit03/bmachine.html).
 
-### Language Description Table
+## Getting Started
+
+### Prerequisites
+The Python 3 interpreter must be installed
+
+### Installation
+Open a terminal and run `git clone https:\\github.com\showalrm\sn-machine`
+
+### Execution
+Run the `__main__` module to execute the program.
+
+## Usage
+Upon execution of the program the user is prompted with the option of choosing the number of register
+cells and the number of memory cells they would like. The user may choose up to 16 register cells
+and up to 256 memory cells.
 
+After configuration, the user may use the following keys to edit the cells.
+
+- `r` - edit the contents of register cells
+- `m` - edit the contents of memory cells
+- `return` - step through the program
+- `i` - edit the instruction counter
+- `e` - execute the program
+- Any other input will quit the program
+
+### Language Description Table
 |Op-Code   | Operand | Description |
 |:--------:|:-------:|-------------|
 |1         | RXY     | LOAD register R with the bits found in memory cell XY |
 |2         | RXY     | LOAD register R with the hex value XY|
 |3         | RXY     | STORE the contents found in register R in the memory cell XY |
 |4         | *RS     | MOVE the bit pattern found in register R to register S | 
-|5         | RST     | ADD the bit patterns in registers S and T and put the result in register R in a two’s complement complement representation |
-|6         | RST     | ADD the bit patterns in registers S and T and put the result in register R in a floating-point notation |
+|5         | RST     | ADD the bit patterns in registers S and T and put the result in register R in a two’s complement complement representation|
+|6         | RST     | ADD the bit patterns in registers S and T and put the result in register R in a floating-point notation (**NOTE** Op-Code 6 does the same as 5)|
```

Comment:
clarify that for now, it is the same operation, and that floating point addition is not available.